clang/include/clang/Driver/Options.td

```diff
@@ -6103,6 +6103,12 @@ def fsycl_enable_int_header_diags: Flag<["-"], "fsycl-enable-int-header-diags">,
   HelpText<"Enable diagnostics that require the SYCL integration header.">,
   MarshallingInfoFlag<LangOpts<"SYCLEnableIntHeaderDiags">>;
 
+// FIXME: the option probably needs better name
+def fsycl_overide_main_fileid : Flag<["-"], "fsycl-override-main-file-id">,
+  HelpText<"Tells compiler that -mail-file-name contains an absolute path and "
```

Comment:
```suggestion
  HelpText<"Tells compiler that -main-file-name contains an absolute path and "
```

Reply:
Fixed in 0a351d1a1b556e14c1dd679bae13ea3e374631dc